examples/scroll/README.md

```diff
@@ -0,0 +1,28 @@
+# Scroll Elements Into View
+
+**Test Code**: [index.js](index.js)  
+
```

Comment:
Need to add the disclaimer:

In most cases, TestCafe will scroll to the target element itself. But in rare cases, you can need to manipulate with scroll manually. You can do it in 2 ways: ...

Reply:
added